packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -293,12 +295,119 @@ The following is an example System Log rate limit event where too many OTP verif
 }
 ```
 
+The following is an example System Log rate limit event where too many profile reload attempts were made for the Active Directory or LDAP agent.
```

Comment:
The following is an example of a System Log rate limit event for too many Active Directory or LDAP agent profile reload attempts

Reply:
fixed.